rust/libnewsboat-ffi/src/utils.rs

```diff
@@ -531,3 +531,15 @@ pub unsafe extern "C" fn rs_strip_comments(line: *const c_char) -> *mut c_char {
         result.into_raw()
     })
 }
+
+#[no_mangle]
+pub unsafe extern "C" fn rs_extract_filter(line: *const c_char, filter: *mut c_char, url: *mut c_char) {
+    abort_on_panic(|| {
+        let line = CStr::from_ptr(line);
+        let line = line.to_str().expect("line contained invalid UTF-8");
```

Comment:
Please add comments to all calls to `expect` and `unwrap`, explaining why panics won't ever happen.

Reply:
This should be the same as the line above. But the function above does not have comments too.

Reply:
Indeed, the function above doesn't have a comment. What's even more embarrassing is that the function was written by me ._. I'll make a PR to add a comment there.

Please add a comment to your function.

Reply:
Okay, I added some comments, not sure if it is correct though.

Reply:
Okay, whatever, I'll amend it in my PR then.